examples/options/include/traccc/options/options.hpp

```diff
@@ -92,7 +91,7 @@ inline std::ostream& operator<<(std::ostream& os,
 
     for (size_t i = 0; i < kSize; ++i) {
         if (0u < i) {
-            os << s_separator;
+            os << s_separator.c_str();
```

Comment:
I think this is a fine change in order to avoid creating a temporary `std::string` every time the function is called. Even if I don't actually care about the performance of the command line processing code...

But not passing the string directly to the output stream as-is, is really just over-doing things. Let's just give the string as-is to the stream. (I.e. don't modify this line.)

Reply:
OK